LiveSplit.AutoSplitters.xml

```diff
@@ -23866,4 +23866,14 @@
         <Type>Script</Type>
         <Description>Auto Splitting and Load Removal available. Compare to IGT. (By TheDementedSalad)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Mortal Kombat Trilogy (GOG &amp; DOSBox)</Game>
```

Comment:
This should be the name of the game on speedrun.com.
```suggestion
            <Game>Mortal Kombat Trilogy</Game>
```

Reply:
All right